pystan/misc.py

```diff
@@ -637,7 +637,12 @@ def _get_valid_stan_args(base_args=None):
 
     args['init_radius'] = args.get('init_r', 2.0)
     if (args['init_radius'] <= 0):
-        args['init'] = "0".encode('ascii')
+        args['init'] = b"0"
```

Comment:
This bit correctly initializes to 0. This was slightly different behavior in Stan for the same args, but doing this will make it the same from PyStan.